### PR TITLE
fix : daily reminders function

### PR DESCRIPTION
**Pull request by Dhruv9449**
moved the hourly reminder code to daily reminder, made a few minor other changes

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -65,12 +65,7 @@ async def stopwach(ctx):
...
-    # Takes input from the user (task) about what they would like to accomplish
-    task = " ".join(task)
-    await ctx.send(ctx.author.mention + f" Task: {task}, saved successfully")
-
-    # Start reminder loop
-    reminder_loop.start(ctx, task)
+    pass
 
 
 @tasks.loop(minutes=60)
```
Comment (mogiiee) [inline bot.py:71]: please refactor the code of the loop to after the daily reminders function

**Review by mogiiee — CHANGES_REQUESTED** [review]
@Dhruv9449 please make the changes in order to get it merged

**Comment by Dhruv9449** [comment]
@mogiiee I've added the loop function below the command function

**Comment by mogiiee** [comment]
gg on the work, LGTM . Thank you fro your contribution.

**Pull request merged**